### DISCUSSION TITLE
Forms of Collaboration > Synchronicity > Concurrent Collaboration

### DISCUSSION THREAD

**Discussion by giscus** [pgpais/live-framework-for-cooperative-games#92] [Framework Discussions]
# Forms of Collaboration > Synchronicity > Concurrent Collaboration

https://www.lfcooperativegames.com/reports/new

**Comment by hg65we** [comment]
Each match happens with all players at the same time and they all act concurrently, meaning there's no order between actions and there is no cooperation between those players outside that match
